certbot/lock.py

```diff
@@ -208,26 +209,30 @@ class _WindowsLockMechanism(_BaseLockMechanism):
     A Windows lock file mechanism.
     By default on Windows, acquiring a file handler gives exclusive access to the process
     and results in an effective lock. However, it is possible to explicitly acquire the
-    file handler in shared access in terms of read and write, and this is done by os.open
+    file handler in shared access in terms of read and write, and this is done by filesystem.open
     and io.open in Python. So an explicit lock needs to be done through the call of
     msvcrt.locking, that will lock the first byte of the file. In theory, it is also
     possible to access a file in shared delete access, allowing other processes to delete an
     opened file. But this needs also to be done explicitly by all processes using the Windows
     low level APIs, and Python does not do it. As of Python 3.7 and below, Python developers
     state that deleting a file opened by a process from another process is not possible with
-    os.open and io.open.
+    filesystem.open and io.open.
```

Comment:
```suggestion
    os.open and io.open.
```
Python devs aren't making statements about filesystem.open

Reply:
Same as above.